.github/workflows/unit_test_pipeline.yml

```diff
@@ -0,0 +1,66 @@
+name: TM1Py Integration Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      environments:
+        description: 'JSON array of environments to test (e.g., ["tm1-11-onprem", "tm1-11-cloud"])'
+        required: true
+        default: '["tm1-11-onprem","tm1-11-cloud","tm1-12-mcsp"]'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment: ${{ fromJson(github.event.inputs.environments) }}
+    environment: ${{ matrix.environment }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          pip install -e .[unit_test]
```

Comment:
`pip install -e .[pandas, dev, unit_test]` would be better. Than you could remove the duplicated packages under *unit_test* in *setup.py*

Reply:
Sounds reasonable to me; I'll modify the yml and setup accordingly. @MariusWirtz - thoughts?

Reply:
Sounds good to me

Reply:
Addressed in #1173